CrudAPI/CrudAPI/Controllers/BankAccountController.cs

```diff
@@ -0,0 +1,83 @@
+﻿using AutoMapper;
+using Azure;
+using CrudAPI.DTOs;
+using CrudAPI.Models;
+using CrudAPI.Models.Database;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.JsonPatch;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace CrudAPI.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class BankAccountController : ControllerBase
+    {
+        private readonly ApplicationDbContext _context;
+        private readonly IMapper _mapper;
+        public BankAccountController(ApplicationDbContext context , IMapper mapper)
+        {
+                _context = context;
+                _mapper = mapper;
+        }
+
+        [HttpPost]
+        [ActionName("Create Account")]
+        public async Task<IActionResult> Create(BankAccountDTO bankAccountDTO)
+        {
+            var  bankAccount = _mapper.Map<BankAccount>(bankAccountDTO);
+
+            await _context.BankAccounts.AddAsync(bankAccount);
+
+            int acn = await _context.SaveChangesAsync();
+
+            if(acn >0)
+                return Ok("User Added Successfully!");
+            return BadRequest("Error While Adding User !");
+        }
+
+        [HttpGet]
+        [ActionName("Get Account")]
+        public async Task<IActionResult> GetAccount(int? id)
+        { 
+            if(id is  null)
+            {
+               var accountList = await _context.BankAccounts.ToListAsync();
+                if (accountList == null)
+                    return BadRequest("Accounts does not exist.");
+                return Ok(accountList);
```

Comment:
Actual Database entities should not be returned. Instead use mapper to map it to DTO Model or View Model